sqlmesh/core/model/definition.py

```diff
@@ -564,6 +564,14 @@ def depends_on_past(self) -> bool:
 
         return self._depends_on_past
 
+    @property
+    def forward_only(self) -> bool:
+        return isinstance(self.kind, _Incremental) and self.kind.forward_only
```

Comment:
add an `is_incremental` property to `ModelKindMixin` which is either `is_incremental_by_time_range` or `is_incremental_by_unique_key`?

Reply:
I think I still need the `isinstance` check to make mypy happy, right?